portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -6,14 +6,39 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Getting rid of the doGet functionality seems problematic since there's now no way to retrieve anything from the data endpoint. Is this intentional?